contracts/0.8.9/ModuleSolo.sol

```diff
@@ -0,0 +1,137 @@
+// SPDX-FileCopyrightText: 2021 Lido <info@lido.fi>
+// SPDX-License-Identifier: GPL-3.0
+
```

Comment:
Maybe move this contract somewhere? It's a bit misleading to have this one on a top level.